ravenframework/Optimizers/GeneticAlgorithm.py

```diff
@@ -419,6 +428,8 @@ def handleInput(self, paramInput):
       self._mutationLocs = None
     else:
       self._mutationLocs = mutationNode.findFirst('locs').value
+    mutationProbNode = mutationNode.findFirst('mutationProb')
+    self._mutationProbTpye = mutationProbNode.parameterValues['type']
     self._mutationProb = mutationNode.findFirst('mutationProb').value
     self._mutationInstance = mutatorsReturnInstance(self,name = self._mutationType)
```

Comment:
This is totally wrong. This is not what I asked. please revise my comments and redo this